bin/compile

```diff
@@ -1,16 +1,28 @@
 #!/bin/bash
 
+mkdir -p $1 $2 $3
 BUILD_PATH=$(cd $1 && pwd)
+CACHE_PATH=$(cd $2 && pwd)
+BUILDPACK_PREFIX=nmap
 
 NMAP_VERSION="7.60"
 if [ -f $3/NMAP_VERSION ]; then
   NMAP_VERSION="$(cat $3/NMAP_VERSION)"
 fi
+NMAP_NAME=nmap-$NMAP_VERSION
```

Comment:
Maybe name it `NMAP_DIR`?

Reply:
I wast thinking about it, and I feel like NMAP_DIR should be the whole path, so I went with NMAP_NAME instead